src/modules/Dashboard/components/TakeNowModal/TakeNowModal.tsx

```diff
@@ -309,10 +380,17 @@ export const useTakeNowModal = ({ dataTestId }: UseTakeNowModalProps) => {
       [],
     );
 
-    if (!activity || !allParticipants || !participantsAndTeamMembers) {
+    if (!activityOrFlow || !allParticipants || !participantsAndTeamMembers) {
       return null;
     }
 
+    let thumbnail: ReactNode = null;
+    if ('activities' in activityOrFlow) {
+      thumbnail = <FlowSummaryThumbnail activities={activityOrFlow.activities} />;
+    } else if ('image' in activityOrFlow) {
```

Comment:
Activities without an image have their `image` property set to an empty string, so the thumbnail breaks

<img width="480" alt="image" src="https://github.com/ChildMindInstitute/mindlogger-admin/assets/14842108/693e7c45-398d-43c1-ad1b-57ea32e4b93c">

```suggestion
    } else if ('image' in activityOrFlow && activityOrFlow.image) {
```